### DISCUSSION TITLE
How to deploy under a sub-path of a domain

### DISCUSSION THREAD

**Discussion by Dav3rs** [adonisjs/core#4788] [Help]
I have an ingress that routes everything on the path "/admin" to my Adonis app, and need the app to know that the asset files should be loaded with that path prefix.

This is an example on nextjs
https://nextjs.org/docs/pages/api-reference/next-config-js/basePath

**Comment by thetutlage** [comment]
Please checkout [Vite's doc](https://docs.adonisjs.com/guides/basics/vite#vite-config-file) and specifically the `assetsUrl` option

**Reply by Dav3rs** [reply]
Doing this the assets work correctly, but the page it's empty (I have ssr disabled btw), it looks that the intertia router doesn't know that I want to use a path prefix, and since it's not on the url that the router expects it has nothing to show. After an F5, the page does a client side render from cache, but it perform an url update to the url without the prefix, where after another F5 it gives a 404 error since there is nothing there really.

What am i missing?
